lib/publiccloud/instance.pm

```diff
@@ -379,6 +379,8 @@ sub wait_for_ssh {
     # Input parameters, see description in above head2 - Parameters section:
     $args{timeout} = get_var('PUBLIC_CLOUD_SSH_TIMEOUT', $args{timeout} // 600);
     $args{wait_stop} //= 0;
+    $args{scan_ssh_host_key} //= 0;
+    record_info('ZZZ', $args{scan_ssh_host_key});
```

Comment:
debug leftover ? I mean you can keep it if you want just find better title ?

Reply:
I will remove them after I do more VRs